src/components/vm/disks/diskAdd.jsx

```diff
@@ -365,511 +359,499 @@ const CustomPath = ({ idPrefix, onValueChanged, device, validationFailed, hideDe
+                    </FormGroup>
+                    {mode === CREATE_NEW && (
+                        <CreateNewDisk idPrefix={`${idPrefix}-new`}
+                                       onValueChanged={onValueChanged}
+                                       storagePoolName={storagePoolName}
+                                       volumeName={diskParams.volumeName}
+                                       size={diskParams.size}
+                                       unit={diskParams.unit}
+                                       format={diskParams.format}
+                                       validationFailed={validationFailed}
+                                       vmStoragePools={storagePools} />
+                    )}
+                    {mode === USE_EXISTING && (
+                        <UseExistingDisk idPrefix={`${idPrefix}-existing`}
+                                         onValueChanged={onValueChanged}
+                                         storagePoolName={storagePoolName}
+                                         existingVolumeName={diskParams.existingVolumeName}
+                                         validationFailed={validationFailed}
+                                         vmStoragePools={storagePools}
+                                         vms={vms}
+                                         vm={vm} />
+                    )}
+                    {mode === CUSTOM_PATH && (
+                        <CustomPath idPrefix={idPrefix}
+                                    onValueChanged={onValueChanged}
+                                    device={diskParams.device}
+                                    validationFailed={validationFailed} />
+                    )}
+                    {vm.persistent &&
+                    <PermanentChange idPrefix={idPrefix}
+                                     permanent={permanent}
+                                     onValueChanged={onValueChanged}
+                                     vm={vm} />}
+                </Form>
+                <AdditionalOptions cacheMode={diskParams.cacheMode}
+                                   device={diskParams.device}
+                                   idPrefix={idPrefix}
+                                   onValueChanged={onValueChanged}
+                                   busType={diskParams.busType}
+                                   serial={diskParams.serial}
+                                   validationFailed={validationFailed}
+                                   supportedDiskBusTypes={supportedDiskBusTypes} />
+            </>
+        );
     }
 
-    onInsertClicked() {
-        const Dialogs = this.context;
-        const close = Dialogs.close;
-        const { vm } = this.state;
-        const { disk } = this.props;
+    return (
+        <Modal position="top" variant="medium" id={`${idPrefix}-dialog-modal-window`} isOpen onClose={Dialogs.close}
+               title={isMediaInsertion ? _("Insert disc media") : _("Add disk")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1303-20231117-090529-31f818c6-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)